app/gateways/api/http/schemas/create_transfer.go

```diff
@@ -0,0 +1,79 @@
+package schemas
+
+import (
+	"time"
+
+	"github.com/jpgsaraceni/suricate-bank/app/domain/entities/account"
+	"github.com/jpgsaraceni/suricate-bank/app/domain/entities/transfer"
+	"github.com/jpgsaraceni/suricate-bank/app/gateways/api/http/responses"
+	"github.com/jpgsaraceni/suricate-bank/app/vos/money"
+)
+
+type CreateTransferRequest struct {
+	AccountDestinationId string `json:"account_destination_id"`
+	Amount               int    `json:"amount"`
+}
+
+type CreateTransferResponse struct {
+	Id                   string    `json:"transfer_id"`
+	AccountOriginId      string    `json:"account_origin_id"`
+	AccountDestinationId string    `json:"account_destination_id"`
+	Amount               string    `json:"amount"`
+	CreatedAt            time.Time `json:"created_at"`
+}
+
+func CreatedTransferToResponse(createdTransfer transfer.Transfer) CreateTransferResponse {
+	return CreateTransferResponse{
+		Id:                   createdTransfer.Id.String(),
+		AccountOriginId:      createdTransfer.AccountOriginId.String(),
+		AccountDestinationId: createdTransfer.AccountDestinationId.String(),
+		Amount:               createdTransfer.Amount.BRL(),
+		CreatedAt:            createdTransfer.CreatedAt,
+	}
+}
+
+func (r CreateTransferRequest) Validate(response *responses.Response, originId account.AccountId) (transfer.Transfer, bool) {
```

Comment:
Receber parametros ponteiro pode ser perigoso pois passamos a responsabilidade de alterar uma instancia para outra func. Costumamos evitar.
O que acha de retornar o bool, já retornar o response. Algo assim:
```suggestion
func (r CreateTransferRequest) Validate(response responses.Response, originId account.AccountId) (transfer.Transfer, responses.Response) {
```